### PR TITLE
Document Update

### PR DESCRIPTION
**Pull request by NiteshKoushik**
Changes from "advice" to "advise"

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/features/datasources/testdata.md [hunk 1]
```diff
@@ -50,5 +50,5 @@ Otherwise the dashboard will not be updated automatically for other Grafana user
 
 ## Using test data in issues
 
-If you post an issue on github regarding time series data or rendering of time series data we strongly advice you to use this data source to replicate the data.
+If you post an issue on github regarding time series data or rendering of time series data, we strongly advise you to use this data source to replicate the data.
```
Comment (achatterjee-grafana) [inline docs/sources/features/datasources/testdata.md:53]: GitHub, not github

**Review by achatterjee-grafana — APPROVED** [review]
Please make the change, otherwise good!

File: docs/sources/features/datasources/testdata.md (hunk 1 above)
Comment (achatterjee-grafana) [reply docs/sources/features/datasources/testdata.md:53]: @NiteshKoushik, can you make the suggested change. I can then merge the PR. Thanks!

Comment (NiteshKoushik) [reply docs/sources/features/datasources/testdata.md]: Hello @achatterjee-grafana , changes made as per suggestion.

**Comment by NiteshKoushik** [comment]
Changes made as per the suggestion.

**Pull request merged**